scripts/scripts.js

```diff
@@ -67,6 +67,11 @@ async function loadFonts() {
   }
 }
 
+export const targetObject = {
+  isMobile: window.matchMedia('(max-width: 767px)').matches,
+  isTab: window.matchMedia('(max-width: 1024px)').matches,
```

Comment:
there isnt confirmation from customer so far 
we have 768px (original site) mapped to 900px for us and
1024px (original site) mapped to 1200px for us

Reply:
done